### PR TITLE
Feature/ipv4

### PR DESCRIPTION
**Pull request by KacperWojciechowski**

### CODE REVIEW

File: src/Internet.cpp
Comment (cieslarmichal) [inline src/Internet.cpp:1]: this file should be in src/modules/internet

File: README.md [hunk 1]
```diff
@@ -100,6 +100,23 @@ Please check [CONTRIBUTING](https://github.com/cieslarmichal/faker-cxx/blob/main
 
 [Discord Channel](https://discord.gg/h2ur8H6mK6) for contributors.
 
+<<<<<<< HEAD
```
Comment (cieslarmichal) [inline README.md:103]: please remove conflict markers

File: include/faker-cxx/Internet.h [hunk 2]
```diff
@@ -132,5 +133,47 @@ class Internet
...
      */
     static unsigned httpStatusCode(std::optional<HttpResponseType> responseType = std::nullopt);
+
+    /**
+     * @brief Determines which address class user wants to generate in the ipv4 method.
+     * 
+     */
+    enum class IPv4Class
+    {
+        classA,
```
Comment (cieslarmichal) [inline include/faker-cxx/Internet.h:143]: consider replacing it with enum values A, B, C (with upper case) as we have word "class" in enum name so it is a duplicate

**Review by cieslarmichal — CHANGES_REQUESTED** [review]
Good job, added couple of comments :)

File: README.md (hunk 1 above)
Comment (KacperWojciechowski) [reply README.md:103]: Done

File: include/faker-cxx/Internet.h (hunk 2 above)
Comment (KacperWojciechowski) [reply include/faker-cxx/Internet.h:143]: Done

File: src/Internet.cpp
Comment (KacperWojciechowski) [reply src/Internet.cpp:1]: Fixed

**Review by KacperWojciechowski — COMMENTED** [review]
Resolved comments

**Review by cieslarmichal — APPROVED** [review]

**Pull request merged**